.azure-pipelines/client.yml

```diff
@@ -34,6 +34,12 @@ jobs:
       scriptPath: 'scripts/devops_tasks/build_packages.py'
       arguments: '-d "$(Build.ArtifactStagingDirectory)" "$(build_targeting_string)"'
 
+  - script: |
```

Comment:
I believe this task should be changed from `script` to `bash`.  The former is only for script code which is valid in both Windows command.com and bash.  I believe `mv` is specific to `bash`.